roles/openshift_docker/tasks/main.yml

```diff
@@ -4,7 +4,7 @@
 # openshift_image_tag correctly for upgrades.
 - name: Set version when containerized
   command: >
-    docker run --rm {{ openshift.common.cli_image }}:latest version
+    docker run --rm {{ openshift.common.cli_image }} version
```

Comment:
This was needed for a docker 1.8.2 bug.  It seems like if you run <some image>:latest explicitly and the latest tag is not already pulled you hit and error about the image ID missing.  If you leave it off it's assumed and will correctly pull the image if needed.